packages/rancher-istio/charts/values.yaml

```diff
@@ -7,47 +7,52 @@ installer:
   tag: 1.7.1-rancher1
 
 istiocoredns:
-    enabled: false
-    image:
-      repository: rancher/coredns-coredns
-      tag: 1.6.2
-    pluginImage:
-      repository: rancher/istio-coredns-plugin
-      tag: 0.2-istio-1.1
+  enabled: false
+  image:
+    repository: rancher/coredns-coredns
+    tag: 1.6.2
+  pluginImage:
+    repository: rancher/istio-coredns-plugin
+    tag: 0.2-istio-1.1
 
 base:
-    enabled: true
+  enabled: true
 
 cni:
-    enabled: false
-    repository: rancher/istio-install-cni
-    tag: 1.7.1
+  enabled: false
+  repository: rancher/istio-install-cni
+  tag: 1.7.1
 
 egressGateways:
-    enabled: false
-    type: NodePort
+  enabled: false
+  type: NodePort
 
 ingressGateways:
-    enabled: true
-    type: NodePort
+  enabled: true
+  type: NodePort
 
 istiodRemote:
-    enabled: false
+  enabled: false
 
 pilot:
-    enabled: true
-    repository: rancher/istio-pilot
-    tag: 1.7.1
+  enabled: true
+  repository: rancher/istio-pilot
+  tag: 1.7.1
 
+#Mixer Policy is deprecated in 1.7.x, will not be supported in 1.8.x 
 policy:
-    enabled: true
-    repository: rancher/istio-mixer
-    tag: 1.7.1
+  enabled: false
+  repository: rancher/istio-mixer
+  tag: 1.7.1
 
 telemetry:
-    enabled: true
```

Comment:
Can you see in istio 1.8 what this is going to be ? My bet is they are removing v1 and v2 and leaving this

Reply:
In the current master, I see that they removed the v1 check in the configmap, but kept the v2 check in the enoyfilter creation. They are tracking mixer removal here https://github.com/istio/istio/issues/25333